### DISCUSSION TITLE
RBD and CEPHFS in a same pool is practically bad?

### DISCUSSION THREAD

**Discussion by caffeinism** [rook/rook#10100] [Q&A] [answered]
Hello! I am new to ceph.

The point of my question is, can't the pool created by cephblockpool be reused in the datapool of cephfilesystem?

I was curious about this, but a search was hard to come up with. Also, rook's cephfilesystem was able to write only the name of the pool in the datapools list, but it didn't really work.

Is this a bad setting?

**Comment by travisn** [comment, marked as answer]
> Hello! I am new to ceph.
> 
> The point of my question is, can't the pool created by cephblockpool be reused in the datapool of cephfilesystem?

No, the pools created by the CephBlockPool and CephFilesystem are separate and cannot be reused.

> I was curious about this, but a search was hard to come up with. Also, rook's cephfilesystem was able to write only the name of the pool in the datapools list, but it didn't really work.
> 
> Is this a bad setting?

The name of the data pools in the CephFilesystem also have a prefix added to the pool names. The prefix is the name of the filesystem. This keeps the pool names unique per filesystem. From the toolbox you should see with `ceph osd pool ls detail` the pools that were created with the prefix.
